mgmt2/rpc/jsonrpc/JsonRPCManager.cc

```diff
@@ -54,9 +59,15 @@ std::condition_variable g_rpcHandlingCompletion;
 ts::Rv<YAML::Node> g_rpcHandlerResponseData;
 bool g_rpcHandlerProccessingCompleted{false};
 
-// jsonrpc log tag.
-static constexpr auto logTag    = "rpc";
-static constexpr auto logTagMsg = "rpc.msg";
+// --- Helpers
+std::pair<ts::Errata, error::RPCErrorCode>
```

Comment:
Isn't pair kinda deprecated is favor of tuple?

Reply:
Meh. The better question is, why `pair` instead of the `struct Error` defined earlier.